source/_integrations/renault.markdown

```diff
@@ -38,6 +38,8 @@ This integration provides the following platforms:
 
 All vehicles linked to the account should then get added as devices, with sensors added as linked entity.
 
+All feature requires at least *Pack EV Remote Control*. For device tracker it requires *Pack Smart Navigation*.
```

Comment:
Not all features.
I think most sensors work regardless.
Switches/Selects/Buttons probable need remote pack....

Also packs differ by country so might need to be more generic.

Reply:
yes I understand. In France, without EV Remote Control you loose all API access and integration is totally broken. I've tested recently and it requires me to subscribe to this pack which is not too expensive. For the tracking I thought it was an issue in the integration. But after reviewing issue tracker I saw it requires another pack that's why I want to warn people in order to not be surprised or looking for something else.